sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -9,15 +9,25 @@
             : '-'
         "
       />
-      <separator v-if="headerDetails.institutionName"></separator>
+      <v-divider
```

Comment:
Very minor, In figma the color is `$brand-grey`. Divider has a property called color and the brand-grey theme is `secondary` in our application. so, you can add `<v-divider color="secondary"`

Reply:
Maybe we can go with this default opacity for vertical, as opacity:1  was too dark for  the vertical divider, when I was doing my story

Reply:
updated